examples/OcclusionSaliency.ipynb

```diff
@@ -155,23 +287,28 @@
     }
    ],
    "source": [
-    "# Generating final saliency map for selected classes\n",
-    "final_saliency_maps = occ_alg.generate(org_preds, prtbd_conf, pertmask)\n",
+    "# Visualize!\n",
+    "import matplotlib.pyplot as plt\n",
     "\n",
-    "sub_plot_ind = len(final_saliency_maps) + 1\n",
+    "sub_plot_ind = len(class_sal_maps) + 1\n",
     "\n",
     "# Visualizing Saliency maps for both classes\n",
-    "plt.figure()\n",
+    "plt.figure(figsize=(16, 4))\n",
     "plt.subplot(1, sub_plot_ind, 1)\n",
-    "plt.imshow(test_img)\n",
+    "plt.imshow(test_image)\n",
     "plt.axis('off')\n",
     "plt.title('Test Image')\n",
     "\n",
-    "for i, class_sal_map in enumerate(final_saliency_maps):\n",
+    "pm_sum = pert_masks.sum(axis=0)\n",
+    "\n",
+    "for i, (class_sal_map, label) in enumerate(zip(class_sal_maps, sal_class_labels)):\n",
     "    plt.subplot(1, sub_plot_ind, i+2)\n",
-    "    plt.imshow(test_img, alpha=0.5)\n",
-    "    plt.imshow(class_sal_map, cmap='jet', alpha=0.3)\n",
-    "    plt.title('{}'.format(sal_classes[i]))\n",
+    "    plt.imshow(test_image, alpha=0.7)\n",
+    "    plt.imshow(\n",
+    "        class_sal_map / pm_sum,\n",
```

Comment:
@vbhavank Pinging you to review this part of the notebook update by request from Brian. I believe that I added this via your direction. Could you confirm that this is appropriate here + why? Brian expressed that the denominator cancels out from the `plt.imshow` auto-normalization.

Reply:
I do believe we should be normalizing with pm_sum. The normalization that is being achieved in the plt.imshow was included [here](https://github.com/Purg/xaitk-saliency/blob/dev/update-notebook-example/xaitk_saliency/utils/masking.py#L115). Therefore, re-normalizing would cancel out.

Reply:
I read two different conclusions there: "we should be normalizing with pm_sum" and "re-normalizing would cancel out".

Reply:
@Purg, I apologize. I meant to say I do *not* believe we should be normalizing pm_sum.

Reply:
OK, sounds good. I will remove this denominator.